.azdo/pipelines/generate-package.yml

```diff
@@ -0,0 +1,101 @@
+trigger:
+  tags:
+    include:
+    - '*'
```

Comment:
So this pipeline expect that a tag is pushed. Wouldn't be nicer to also generate and push a tag in the pipeline? And the pipeline would just expect a version parameter from a user, and not that a user pushes the tag.

Reply:
made tagging part of the pipeline